wpcli-dev/provision.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Provision WP-CLI develop and do an alias to the original wpcli
+
+cd "/srv/www/"
```

Comment:
The utilities folder itself is accessible inside the VM, I don't see why we need to put this in `/srv/www`

Reply:
Well if someone want to contribute is more easy to access on `www` folder then look for the utilities one in the host machine.
Also considering that I did mainly for consistency and because a lot of other utilities are inside the `default` folder that usually no ones touch.

Reply:
Those utilities install things that can be opened in the browser, I'd prefer if it stayed in the utilities folder

Reply:
hmm it would be easier for people to use though, so I'm going to keep this